DOCS/contribute.md

```diff
@@ -120,10 +120,10 @@ Touching user-visible parts may require updating the mpv docs
 -------------------------------------------------------------
 
 - Most user-visible things are normally documented in DOCS/man/. If your commit
-  touches documented behavior, list of sub-options, etc., you need to adjust the
+  touches documented behavior, a list of sub-options, etc., you need to adjust the
   documentation.
 - These changes usually go into the same commit that changes the code.
-- Changes to command line options (addition/modification/removal) must be
+- Changes to command-line options (addition/modification/removal) must be
```

Comment:
"command line" as two words is a lot more common and more well understood. However, the hyphen isn't *technically* wrong, I guess. Even wikipedia has the hyphen in their article.